src/generate/azgenerator.ts

```diff
@@ -64,15 +65,22 @@ export async function processRequest(host: Host) {
                     f.endsWith('setup.cfg') ||
                     // f.endsWith('report.md') ||
                     f.endsWith('tests/__init__.py') ||
-                    f.endsWith('preparers.py')
+                    f.endsWith('preparers.py') ||
+                    f.endsWith('commands.py')
                 ) {
                     host.WriteFile(f, files[f]);
+                    if (f.endsWith('commands.py')) {
+                        tolintFiles.push(path.join(model.azOutputFolder, f));
+                    }
                 } else {
                     host.WriteFile(f, files[f].join(EOL));
                 }
             }
         }
         closeInplaceGen();
+        // tolintFiles.forEach(async (fileName) => {
```

Comment:
do we need this?

Reply:
no, we don't need tolintFiles, This is from previous version, I forget to remove it.